db/blip_handler.go

```diff
@@ -457,6 +457,18 @@ func (bh *blipHandler) sendChanges(sender *blip.Sender, opts *sendChangesOptions
 		return false
 
 	}
+	auditFields := base.AuditFields{
+		base.AuditFieldSince:    options.Since.String(),
+		base.AuditFieldChannels: channelSet,
```

Comment:
AuditFieldChannels and AuditFieldDocIDs ares optional, should only be populated when non-empty.

If we're trying to align with the REST-based events, we should populated AuditFieldFilter based on whether channels or docIDs are empty.

Reply:
Agreed, I thought erroneously that the way this marshaled it would omit empty fields. I've added testing for this.